.github/workflows/release.yml

```diff
@@ -1,10 +1,10 @@
 name: Build and upload to PyPI
 
-# Only build on tagged releases
+# Only build on published releases
 on:
-  push:
-    tags:
-      - '*'
+  release:
+    types:
+      - published
```

Comment:
Do we need to add a release guide to explain what we need to do to trigger this? I assume it is "go to GitHub and publish a release" because tagging via command line and pushing a tag is not good enough here?

Reply:
I mean, we can stick with "tag and push" if we want, I just looked and saw we were already using the GitHub releases feature.

Reply:
I'm good with this, I've been making releases through the web interface.